Uzduotis05/Program.cs

```diff
@@ -0,0 +1,104 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Uzduotis05
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            /*
+             * Išveskite į ekraną dviejų skaičių sumą, skirtumą, sandaugą ir dalmenį,
+             * kiekvieną kartą nurodant atliekamą veiksmą ir šių skaičių reikšmes.
+             * Pvz.: 
+             * 4 + 5 = 9
+             * 4 - 5 = -1
+             * …
+             */
+
+            int skaicius1 = 4;
+            int skaicius2 = 5;
+            float skaicius3 = 4;
+            float skaicius4 = 5;
```

Comment:
Manau, kad čia galima apsieiti ir be float kintamųjų, kadangi realiai jie reikalingi tik dalybos operacijos. Dalybos operaciją galite atlikti vieną iš kintamųjų paverčiant į float pačioje operacijoje: 
`Console.WriteLine($"{skaicius1} / {skaicius2} = {(float)skaicius1 / skaicius2} ");`

Be to, saugant reikšmę `float` tipo kintamajame, reikia prie reikšmės pridėti `f` simbolį:
`float skaicius3 = 4f;`